### DISCUSSION TITLE
Just an idea if its possible to have a date time picker instead of select dropdown.

### DISCUSSION THREAD

**Discussion by tnchuntic** [mattbanks/woocommerce-local-pickup-time#163] [Ideas]
Just an idea if it's possible to have a date-time picker instead of a select dropdown.

**Comment by timnolte** [comment]
Yes, this has been requested before, however there are multiple challenges with attempting to do that. One problem is that date/time pickers are an accessibility nightmare. The biggest problem is being able to set date & time restrictions, with a date/time picker this becomes very complicated and would require a JavaScript solution.

**Comment by timnolte** [comment]
There is an existing request open here. https://github.com/WC-Local-Pickup/woocommerce-local-pickup-time/issues/127
